### PR TITLE
Add version checking workflow

### PR DESCRIPTION
**Pull request by szepeviktor**
Compares tagged version to `Root::VERSION`.

### CODE REVIEW

File: .github/workflows/git-tags.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                extensions: "dom, curl, libxml, mbstring, zip, pcntl, pdo, sqlite, pdo_sqlite, gd, fileinfo, exif"
+        -
+            name: "Checkout repository"
+            uses: "actions/checkout@v4"
+            with:
+                fetch-depth: 0
+        -
+            name: "Check latest tagged version"
+            run: |
+                LATEST_TAG="$(git describe --tags --abbrev=0)"
```
Comment (szepeviktor) [inline .github/workflows/git-tags.yml:36]: I'd like to send another PR with
```bash
LATEST_TAG="${{ github.ref_name }}"
```

**Comment by iamgergo** [comment]
Can you explain how does it work exactly?

**Comment by szepeviktor** [comment]
```
LATEST_TAG="$(git describe --tags --abbrev=0)"
```
Reads the latest git tag.

```
CURRENT_VERSION="v$(php -r 'require __DIR__."/src/Root.php"; echo Cone\Root\Root::VERSION;')"
```
Reads `Root::VERSION` constant.

Then errors out if the two are not the same.
